MapboxDirectionsTests/V5Tests.swift

```diff
@@ -86,6 +86,18 @@ class V5Tests: XCTestCase {
         XCTAssertEqual(round(coordinate.longitude), -122)
         
         XCTAssertEqual(leg.steps[18].name, "Sycamore Valley Road West")
+        
+        let intersection = step.intersections![0]
```

Comment:
You can also say `step.intersections.first!`. I find that easier to type because of code completion.